### PR TITLE
Build script and documentation

### PR DESCRIPTION
**Pull request by justinjams**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+if [ "$(uname)" == "Darwin" ]; then
+				zip -r moonshot.nw source/*
```
Comment (mildmojo) [inline build.sh]: These need to `cd source` before zipping so they don't store the `source` dir in the zip. Node-webkit wants the zip to have `index.html` at the root level.

**Comment by mildmojo** [comment]
Looks good to me. Tweak the build script and I'll merge it in.

**Comment by justinjams** [comment]
Of course! Thanks for the explanation. It should be all set now!

**Comment by mildmojo** [comment]
Thanks! :cocktail:

**Pull request merged**